draft-ietf-mops-streaming-opcons.md

```diff
@@ -331,15 +331,14 @@ important to consider the effects of network design decisions
 on application-level performance, with considerations for
 the impact on video delivery.
 
-This document aims to provide a taxonomy of networking issues as
-they relate to quality of experience in internet video delivery.
-The focus is on capturing characteristics of video delivery that
-have surprised network designers or transport experts without
-specific video expertise, since these highlight key differences
-between common assumptions in existing networking documents and
-observations of video delivery issues in practice.
-
-Making specific recommendations for mitigating these issues
+This document
+focuses on capturing characteristics of video delivery that have
+surprised network designers or transport experts without specific
+video expertise, since these highlight key differences between common
+assumptions in existing networking documents and observations of
+video delivery issues in practice.
+
+Making specific recommendations for future work on mitigating these issues
```

Comment:
This changes the original intent, and although I agree we're not making recommendations on future work (which might be worth a mention), we're also not making recommendations on operational practices (which was the original intent).

Maybe "Making specific recommendations on operational practices or future work to mitigate these issues is out of scope..."?

Reply:
@GrumpyOldTroll, my aplogies here. I think the discussion about that aspect of this document dates back to https://datatracker.ietf.org/doc/html/draft-jholland-mops-taxonomy-00, and I probably missed some of the discussion. I'll back off part of my suggested change and remove the proposed mention of future work, which is actually mentioned later in that paragraph. 

So, now "Making specific recommendations on operational practices aimed at mitigating these issues is out of scope, though some existing mitigations are mentioned in passing.  The intent is to provide a point of reference for future solution proposals to use in describing how new technologies address or avoid these existing observed problems."